.changeset/popular-ads-visit.md

```diff
@@ -0,0 +1,44 @@
+---
+"@hashicorp/flight-icons": patch
```

Comment:
```suggestion
"@hashicorp/flight-icons": minor
```

minor since we are adding new icons (i'll see if i can make this the default for this action)

Reply:
ahh, this was added manually. @cveigt I think this is always going to be `minor` for the icon requests.

Reply:
for future context, `patch` [is hardcoded](https://github.com/changesets/bot/blob/44541221181b771ba5ed5a37f1f4b28992e14dd8/index.ts#L94)